request.go

```diff
@@ -247,3 +247,25 @@ type RefundReq struct {
 	Amount    int64  `json:"amount"`
 	Reason    string `json:"reason"`
 }
+
+// Subscribe : Represent Subscribe object payload (request and response)
+type Subscribe struct {
+	ID          string      `json:"id,omitempty"`
+	Name        string      `json:"name"`
+	Amount      string      `json:"amount"`
+	Currency    string      `json:"currency"`
+	CreatedAt   string      `json:"created_at,omitempty"`
+	Status      string      `json:"status,omitempty"`
+	Token       string      `json:"token"`
+	PaymentType PaymentType `json:"payment_type"`
+	Schedule    Schedule    `json:"schedule"`
```

Comment:
`ID,CreatedAt,Status` does not seems to be available as request: https://api-docs.midtrans.com/#create-subscription

Reply:
same with answer above, cause it also used as response

Reply:
Let's separate request and response like comment https://github.com/veritrans/go-midtrans/pull/55#discussion_r361421651